pyairtable/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.4.0"
```

Comment:
I guess that does not belong to this PR/feature

Reply:
will revert then